website/docs/r/tenant_policies_mld_snooping_policy.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: "mso"
+page_title: "MSO: mso_tenant_policies_mld_snooping_policy"
+sidebar_current: "docs-mso-resource-tenant_policies_mld_snooping_policy"
+description: |-
+  Manages MLD Snooping Policies on Cisco Nexus Dashboard Orchestrator (NDO)
+---
+
+# mso_tenant_policies_mld_snooping_policy #
+
+Manages Multicast Listener Discovery (MLD) Snooping Policies on Cisco Nexus Dashboard Orchestrator (NDO). This resource is supported in NDO v4.3 and higher.
+
+## GUI Information ##
+
+* `Location` - Manage -> Tenant Template -> Tenant Policies -> MLD Snooping Policy
+
+## Example Usage ##
+
+```hcl
+resource "mso_tenant_policies_mld_snooping_policy" "mld_policy" {
+  template_id                = mso_template.template_tenant.id
+  name                       = "test_mld_snooping_policy"
+  description                = "Test MLD Snooping Policy"
+  admin_state                = "enabled"
+  fast_leave_control         = true
+  querier_control            = true
+  querier_version            = "v2"
+  query_interval             = 125
+  query_response_interval    = 10
+  last_member_query_interval = 1
+  start_query_interval       = 31
+  start_query_count          = 2
+}
+```
+
+## Argument Reference ##
+
+`template_id` - (Required) The unique ID of the tenant policy template.
+`name` - (Required) The name of the MLD Snooping Policy.
+`description` - (Optional) The description of the MLD Snooping Policy.
+`admin_state` - (Optional) The administrative state of the MLD Snooping Policy. Valid values: `enabled`, `disabled`.
```

Comment:
Add default value if the value is available and do the same for other attributes.